SimplenoteUITests/UIDs.swift

```diff
@@ -77,6 +79,10 @@ enum UID {
     enum SearchField {
         static let search = "Search notes or tags"
     }
+
+    enum ContextMenuItem {
+        static let paste = "Paste"
+    }
```

Comment:
created new enum for context menu, figured it would be better to have it's own instead of storing it with buttons.